Liroy/bootstrap.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#Step 1 - Update CentOS 7 system
+echo "Updating system"
+sudo yum -y install epel-release
+sudo yum -y update
+sudo yum install -y yum-utils
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+
+#Install Docker
+echo "Installing Docker"
+sudo yum -y install docker docker-ce docker-ce-cli containerd.io
+which docker
+if [[ $? -eq 0 ]]; then
+  echo "Docker installation succeeded"
+else
+  echo "Docker installation failed"
+fi
+
+#For Jenkins Installation:
+#Install Java:
+echo "Installing java..."
+sudo yum -y install java-1.8.0-openjdk.x86_64
+#Set environment variables:
+echo "Adding environment variables"
+sudo cp /etc/profile /etc/profile_backup #Backup /etc/profile
```

Comment:
What happens to the backup file if you invoke the script twice?

Reply:
I can see now that it would be overwritten. But how can I resolve this? 
The only way I can think about is removing this line from the script, but then it's a risk not to back up the file before running the script.

Reply:
Renaming is a good option. You might want to name the file with a unique name. There are techniques how to come out with a unique name. Look for it.